module/builder/consensus/builder.go

```diff
@@ -395,11 +396,9 @@ func (b *Builder) getInsertableSeals(parentID flow.Identifier) ([]*flow.Seal, er
 			// ATTENTION:
 			// Here, IncorporatedBlockID (the first argument) should be set to
 			// ancestorID, because that is the block that contains the
-			// ExecutionResult. However, in phase 2 of the sealing roadmap, we
-			// are still using a temporary sealing logic where the
-			// IncorporatedBlockID is expected to be the result's block ID.
+			// ExecutionResult.
```

Comment:
I think we can remove this comment
```golang
// ATTENTION:
// Here, IncorporatedBlockID (the first argument) should be set to
// ancestorID, because that is the block that contains the
// ExecutionResult.
```